File: VB/MyBandedGridView/MyBandedGridView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.BandedGrid
Imports DevExpress.XtraGrid

Namespace MyXtraGrid
	Public Class MyBandedGridView
		Inherits BandedGridView
		Public Sub New(ByVal ownerGrid As DevExpress.XtraGrid.GridControl)
			MyBase.New(ownerGrid)

		End Sub
		Public Sub New()

		End Sub
		Protected Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyBandedGridView"
			End Get
		End Property
	End Class
End Namespace

File: CS/MyBandedGridView/MyBandedGridView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.BandedGrid;
using DevExpress.XtraGrid;

namespace MyXtraGrid
{
    public class MyBandedGridView : BandedGridView
    {
        public MyBandedGridView(DevExpress.XtraGrid.GridControl ownerGrid)
            : base(ownerGrid)
        {
            
        }
        public MyBandedGridView()
        {
            
        }
        protected override string ViewName { get { return "MyBandedGridView"; } }
    }
}
